### ISSUE TITLE
flc:  need to iterate xml format and values

### ISSUE THREAD

**Issue by donalgrant** [Caltech-IPAC/Kepler-Discoveries#7]
1. In David's latest files,
inclination is based on 0 being in line with us,
e.g. with Kepler-11g
`<inclination>0.00226892802759271</inclination>`
...but in a working file,
inclination is based on π/2 being in line with us,
`<inclination>1.56492</inclination>`

2. In David's latest files,
it appears that transit duration is in days,
e.g. with Kepler-11g
`<transitDuration>0.3945000</transitDuration>`
whereas in a working file, I think duration is in hours
`<transitDuration>9.516</transitDuration>`

3. In the additional planets,
there are extraneous parameters, not needed.
E.g., a working file has
```
<name>Kepler-11b</name>
<semimajorAxis>0.092</semimajorAxis>
<radius>0.185032</radius>
<period>10.303989</period>
<inclination>1.54971</inclination>
<eccentricity>0.00</eccentricity>
</planet>
```

Whereas in David's latest version it's
```
<name>Kepler-11b</name>
<semimajorAxis>0.091000</semimajorAxis>
<radius>0.161</radius>
<period>10.30390000</period>
<longitudeOfAscendingNode>1.5707963267949</longitudeOfAscendingNode>
<argumentOfPericenter>1.5707963267949</argumentOfPericenter>
<inclination>0.00628318530717964</inclination>
<eccentricity>0.045000</eccentricity>
<transitDuration>0.1715000</transitDuration>
<meanAnomalyAtTransitMiddle>0</meanAnomalyAtTransitMiddle>
</planet>
```

So, in other words,
for the additional planets in the system,
leave out
```
<longitudeOfAscendingNode>
<argumentOfPericenter>
<transitDuration>
<meanAnomalyAtTransitMiddle>
```

**Comment by donalgrant** [comment]
Code updated -- see commit 97c19d7cfc5cfe8ac99ee7c40f754020096ec6e9.
